Formula/a/apigeecli.rb

```diff
@@ -0,0 +1,40 @@
+class Apigeecli < Formula
+  desc "Apigee management API command-line interface"
+  homepage "https://cloud.google.com/apigee/docs"
+  url "https://github.com/apigee/apigeecli.git",
+    tag:      "v2.13.0",
+    revision: "01f1372ad588d9e92d30a4bd5ca32af55b1a7100"
+  license "Apache-2.0"
+  head "https://github.com/apigee/apigeecli.git", branch: "main"
+  # version "2.13.0"
+
+  livecheck do
+    url :stable
+    regex(/^v(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0" # Disable CGO
```

Comment:
```suggestion
```

Reply:
Hi @chenrui333 can you elaborate why we should enable CGO? This program is pure Go, and the  upstream package disables CGO.